Scout/AppDelegate.swift

```
@@ -7,18 +7,21 @@
 //
 
 import UIKit
-
+import GoogleMaps
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
   
   var window: UIWindow?
   
   func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-    BuddyBuildSDK.setup()
     if !PrivateConfig.initConfig() {
       print("PrivateConfig.plist is missing.")
       return false
```

@redphx I think we should let the app crash here.